packages/edit-site/src/components/global-styles/utils.js

```diff
@@ -49,6 +49,12 @@ export const PRESET_METADATA = [
 			},
 		],
 	},
+	{
+		path: [ 'color', 'duotone' ],
+		cssVarInfix: 'duotone',
+		valueFunc: ( { slug } ) => `url( '#wp-duotone-${ slug }' )`,
+		svgFilter: 'duotone',
```

Comment:
Adding `svgFilter` to the `PRESET_METADATA` is how I'm limiting the duotone `editor.PresetSvgFilter` hook to only apply for duotone SVG filters. I'm curious if folks think this is the best way to go about that or if there's a better solution.

Reply:
Do we still need this if we don't use a filter? [related convo](https://github.com/WordPress/gutenberg/pull/37727/files#r840444001).